docs/fips_support.md

```diff
@@ -0,0 +1,45 @@
+# Rancher Kubernetes Enginer 2
+
+## Support for FIPS Cryptography
+
+Starting with Rancher Kubernetes Enginer version 2 (RKE2), you can install a cluster that uses FIPS validated libraries. 
+
+## Use of FIPS Compatible Go compiler.
+
+The Go compiler in use can be found [here](https://hub.docker.com/u/goboring). Each component of the system is built with the version of this compiler that matches the same standard Go compiler version that would be used otherwise. 
+
+
+### FIPS Support in Cluster Components
+
+Most of the components of the RKE2 system are statically compiled with the GoBoring Go compiler implementation that takes advantage of the BoringSSL library. RKE2, from a component perspective, is broken up in a number of sections. The list below contains the sections and associated components.
+
+* etcd
+
+* Kubernetes
+  * API Server
+  * Controller Manager
+  * Scheduler
+  * Kubelet
+  * KubeProxy
```

Comment:
We should mention i tihnk that this particular component (kubeproxy) is separated in its own image

Reply:
Does that matter in this context?

Reply:
I guess not, resolving this one, thanks!